storage/client/queries.go

```diff
@@ -241,7 +241,7 @@ func (qf QueryFactory) ValidatorDataQuery() string {
 					WHERE %[1]s.entities.id = %[1]s.nodes.entity_id
 						AND %[1]s.nodes.roles like '%%validator%%'
 				)
-			WHERE %[1]s.entities.address = $1::text`, qf.chainID)
+			WHERE %[1]s.entities.id = $1::text`, qf.chainID)
```

Comment:
I actually think that we should support fetching entities by address instead, but it's out of scope for this pr. Added for discussion in the product meeting.

Reply:
:+1: I'm fine with whatever the frontend needs. This was just to reflect the current API.